docs/README.md

```diff
@@ -7,8 +7,6 @@
 ## Architecture
 
 * [Architecture Overview](./architecture.md)
-* [Authentication and Authorization of API Keys](./tutorials/roles-management.md)
```

Comment:
Why are the sections removed? Where are they moved to?

Reply:
to corresponding sections in Auth and Access Control